css/style.css

```diff
@@ -358,14 +396,21 @@ section#about-me div button {
     word-spacing: normal;
     letter-spacing: 1.2;
     transition: all 0.3 linear;
+    margin: 5px 0 0 0;
+    padding: 10px 30px;
+    line-height: 1.4rem;
+    text-transform: uppercase;
+    width: 33%;
 }
 
 #contact .contact-about form .form-about button[type='submit']:hover {
     background-color: #e2e2e2;
     color: #1d3f5d;
     border: 1px solid #1d3f5d;
 }
-
+::placeholder {
```

Comment:
In dem Fall ist es okay, weil wir nur ein Formular-Element haben. Aber grundsätzlich ist zu bedenken, dass wir gegebenenfalls die Designs verschiedener Formulare im CSS beschreiben wollen. 
Damit hast Du ja auch angefangen, etwa in dem Du spezifizierst:

```
#contact .contact-about form .form-about button[type='submit']:hover {
...
}
```

Du sagst also, dass die Style-Angaben nur für dieses eine Formular im Container mit id contact und class "contact form" & "form-about" gelten sollen.

Deine Placeholder-Angabe folgt dieser Regel nicht. Alle placeholder in Deinem Projekt würden nun groß geschrieben werden.
Um also Deiner schon angewandten Logik gerecht zu werden (es für dieses eine Formular zu spezifizieren), sähe es etwa so aus:
```
#contact .contact-about form .form-about ::placeholder {
    text-transform: uppercase;
}
```